src/pages/supplements/Supplement.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react"; 
+import "./Supplement.css";
+import { Button, Dropdown, DropdownToggle, DropdownMenu, DropdownItem, Table } from "reactstrap";
+import Sofs from "./Sofs";
+import TravelInsurance from "./TravelInsurance"; 
+
+const SupplementOptions = {
```

Comment:
could you export this? if I can import the `SupplementOptions` into my review page, it will be easier for me to display it in a human readable way

Reply:
I can add an export clause but I'm not sure it'll work properly as is outside of the Supplement component. Would you be able to pass in a list of supplements as a prop?

Reply:
I think it should work if you just wrote it like this: 
```
export const Supplements = () => {
...
}
```

I just want it so I can get the strings and so we don't have to repeat declare this object in multiple places, since it's just an object it should be okay I think. Let me know if I'm not understanding something

Reply:
Oh! That's my bad lol, for some reason I thought you were asking me to export the list of added supplements rather than the list of options. Ope.

Reply:
thx bb